Lib/poplib.py

```diff
@@ -457,7 +441,7 @@ def _create_socket(self, timeout):
                                             server_hostname=self.host)
             return sock
 
-        def stls(self, keyfile=None, certfile=None, context=None):
+        def stls(self, context=None):
```

Comment:
Make `context` keyword-only.

Reply:
Oops, this change got lost in a rebase or another mistake. I did the change again.